packages/storybook/types.d.ts

```diff
@@ -0,0 +1,58 @@
+import { HTMLAttributes, Key } from 'react';
+import { Components as WCTypes } from '@department-of-veterans-affairs/component-library/dist/components/types';
+
+// Applies the component's type plus HTMLAttributes<HTMLElement> from React in order to satisfy the JSX "children" prop
+
+type IsElement = HTMLAttributes<HTMLElement>;
```

Comment:
This `IsElement` type adds standard HTMLElement attributes like id and className and such to the Web Components.